AudioKit/Common/Nodes/AKNode.swift

```diff
@@ -46,6 +46,7 @@ extension AVAudioConnectionPoint {
     }
     
     deinit {
+        //AKLog("* AKNode")
```

Comment:
I'm see a lot of these added commented out prints/logs.  Can't they just be comments?  Were they just around for debug?  Not a big deal, just thought I'd bring it up.  (also, lets always use AKLog and not print so it can be suppressed more easily).